src/main/scala/mesosphere/marathon/core/deployment/impl/AppStartActor.scala

```diff
@@ -9,17 +9,16 @@ import mesosphere.marathon.core.event.DeploymentStatus
 import mesosphere.marathon.core.instance.Instance
 import mesosphere.marathon.core.launchqueue.LaunchQueue
 import mesosphere.marathon.core.readiness.ReadinessCheckExecutor
-import mesosphere.marathon.core.task.tracker.InstanceTracker
 import mesosphere.marathon.state.RunSpec
 
 import scala.concurrent.{Future, Promise}
 
 class AppStartActor(
     val deploymentManagerActor: ActorRef,
     val status: DeploymentStatus,
-    val scheduler: SchedulerActions,
+    val schedulerActions: SchedulerActions,
```

Comment:
I don't see a change to `mesosphere.marathon.SchedulerActions` to use the scheduler. Wdyt about changing it too?

Reply:
Most modules take `scheduler` and `schedulerActions`. The later was called `scheduler` before. Overall `scheduler` might not be a good name because it could be confused with the Mesos scheduler interface.